app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms = @guest.rooms
```

Comment:
Keep in mind that you don't necessarily need to create the @rooms instance var as you can reference it in the view as @guest.rooms. There are Rails-ists who argue that minimizing instance variables is best practice to make sure that your controllers are adhering to SRP principles (Sandi Metz is one of these Rails experts)

This is a for-fun comment, not affecting your grade

Reply:
Nope, in your view whenever you need to access rooms, you would just access them using the `@guest` instance variable
i.e. <% @guest.rooms.each %>

Line 3 here should remain the same - you retrieve the guest by the ID